engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -1276,3 +1359,412 @@ definitions:
+          recommended, though not required. The identifier should attempt to
+          be globally unique so it can be used in downstream aggregator
+          services e.g. Service Registry.
+        example: org.ga4gh.myservice
+      name:
+        type: string
+        description: Name of this service. Should be human readable.
+        example: My project
+      type:
+        $ref: '#/definitions/ServiceType'
+      description:
+        type: string
+        description: >-
+          Description of the service. Should be human readable and provide
+          information about the service.
+        example: This service provides...
+      organization:
+        type: object
+        description: Organization providing the service
+        required:
+          - name
+          - url
+        properties:
+          name:
+            type: string
+            description: Name of the organization responsible for the service
+            example: My organization
+          url:
+            type: string
+            format: uri
+            description: URL of the website of the organization (RFC 3986 format)
+            example: 'https://example.com'
+      contactUrl:
+        type: string
+        format: uri
+        description: >-
+          URL of the contact for the provider of this service, e.g. a link to
+          a contact form (RFC 3986 format), or an email (RFC 2368 format).
+        example: 'mailto:support@example.com'
+      documentationUrl:
+        type: string
+        format: uri
+        description: >-
+          URL of the documentation of this service (RFC 3986 format). This
+          should help someone learn how to use your service, including any
+          specifics required to access data, e.g. authentication.
+        example: 'https://docs.myservice.example.com'
+      createdAt:
+        type: string
+        format: date-time
+        description: >-
+          Timestamp describing when the service was first deployed and
+          available (RFC 3339 format)
+        example: '2019-06-04T12:58:19Z'
+      updatedAt:
+        type: string
+        format: date-time
+        description: >-
+          Timestamp describing when the service was last updated (RFC 3339
+          format)
+        example: '2019-06-04T12:58:19Z'
+      environment:
+        type: string
+        description: >-
+          Environment the service is running in. Use this to distinguish
+          between production, development and testing/staging deployments.
+          Suggested values are prod, test, dev, staging. However this is
+          advised and not enforced.
+        example: test
+      version:
+        type: string
+        description: >-
+          Version of the service being described. Semantic versioning is
+          recommended, but other identifiers, such as dates or commit hashes,
+          are also allowed. The version should be changed whenever the service
+          is updated.
+        example: 1.0.0
```

Comment:
This funny red bar symbol means "end of file". It's a team convention to add a trailing newline at the end of our files to avoid this showing up (and so that files print out nicer on the terminal, etc). I think there's an intellij option to automatically add newlines whenever you edit files (might need to google to find out how to turn it on)